requirements.txt

```diff
@@ -30,7 +30,7 @@ peepdf-fork==0.4.3
 pdfid==1.0.4
 pefile-fork==2020.12.16
 psycopg2-binary==2.9.1
-pycti==5.1.2a0
+pycti==5.1.3
```

Comment:
pls revert these 2 changes.

Reply:
Ok. I was actually trying to undo those changes in my commit (that's why the force push). I added that because it wasn't building on my machine.

Will be removed sir :) 👍🏼